magic-methods/code_stego/code_stego/constants.py

```diff
@@ -0,0 +1,10 @@
+"""Global constants"""
+
+MAGIC_NUMBER = 57764
+USED_BITS_PER_BYTE = 4
+USED_BITS_MASK = 0b1111
```

Comment:
Would like to see commends on each of these variables. After reading the code for a while and learning about Steganography, I was finally able to figure out that USED_BITES_PER_BYTE is the number LSBs in each Byte.